src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/organisations/OrganisationsSynchronisationService.kt

```diff
@@ -323,6 +330,63 @@ class OrganisationsSynchronisationService(
   }
 
   suspend fun corporateInternetAddressInserted(event: CorporateInternetAddressEvent) {
+    val telemetry = telemetryOf("nomisCorporateId" to event.corporateId, "dpsOrganisationId" to event.corporateId, "nomisInternetAddressId" to event.internetAddressId)
+    if (event.doesOriginateInDps()) {
+      telemetryClient.trackEvent(
+        "organisations-internet-address-synchronisation-created-skipped",
+        telemetry,
+      )
+    } else {
+      val nomisCorporate = nomisApiService.getCorporateOrganisation(nomisCorporateId = event.corporateId)
+      val nomisInternetAddress = nomisCorporate.internetAddresses.find { it.id == event.internetAddressId }!!
+      if (nomisInternetAddress.type == "EMAIL") {
```

Comment:
we have to get the Internet address and then decide how to treat it.
TODO Change trigger to include this is event to improve readability